packages/design-system-web/src/components/geov-entity-download-rdf/geov-entity-download-rdf.tsx

```diff
@@ -73,18 +77,22 @@ export class GeovEntityDownloadRdf {
   modal: HTMLIonModalElement;
 
   open() {
-    this.modal.isOpen = true;
+    this.modal.present();
   }
   dismiss() {
     this.modal.dismiss();
-    this.modal.isOpen = false;
   }
+
   async fetchRDF(format: string[]) {
     const headers = new Headers({
       Accept: format[0], //format[0] = Type
     });
+    let args = this.entityId;
+    if (this.projectId != undefined) {
```

Comment:
In toolbox-streams we use project 0 to identify the community data. To avoid confusions, I prefer to use `if(this.projectId !== undefined)` with two equal signs. This way the component even behaves correctly, if we publish a sparql endpoint with project number 0.

Reply:
Done with if(this.projectId !== undefined)